### ISSUE TITLE
Color-Inversion Alternation Flickering 

### ISSUE THREAD

**Issue by MyraTYVT** [dortania/OpenCore-Legacy-Patcher#818]
### Machine Model

iMac12,2

### Application Version

Latest Release

### Application Variant

TUI (Text User Interface)

### Have you verified whether this issue has been opened before? If no, your issue will be closed

true

### Have you verified whether this issue is covered in our guide? ie. Troubleshooting and Legacy Acceleration pages

true

### Is this issue with a 3rd party application?

false

### What versions of macOS are you seeing the problem on?

macOS 11, Big Sur

### Where does this issue happen?

Between booting macOS and Login Screen, Within macOS (Before Logging in), Within macOS (After Logging in)

### What is the Issue?

At the beginning of the login the Mac starts to flickering whenever move the mouse, use the keyboard or even when it loads internet pages or apps. 
I tried to reinstall everything. But the issues stays there. 
I searched online but I didn’t find nothing about it, I don’t know if I just missed it or else. 
Is it fixable? 
thank you.

https://user-images.githubusercontent.com/96150862/146400053-124eaf50-06e9-424f-a1a3-37820b977d34.MOV

**Comment by educovas** [comment]
**Have you verified whether this issue is covered in our guide? ie. Troubleshooting and Legacy Acceleration pages**

> true

https://dortania.github.io/OpenCore-Legacy-Patcher/ACCEL.html#erratic-colours-on-ati-terascale-2-gpus-hd5000-hd6000

**Comment by MyraTYVT** [comment]
@educovas yes i did. Also I tried what on the guide is written. But, unfortunately for me, it did not work.
